srcpkgs/BAND/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'BAND'
+pkgname=BAND
+version=1.10.9
+revision=1
+archs="x86_64"
+hostmakedepends="binutils tar xz gzip"
+depends="gtk+3 libnotify nss libXScrnSaver libXtst xdg-utils at-spi2-core libuuid libappindicator libsecret"
+short_desc="Group Chat Client for Sports, School, Campus Groups, and more"
+maintainer="Adam Pschorr <adampschorr13@aol.com>"
+license="custom:Proprietary"
+homepage="https://band.us/home"
+distfiles="https://ssl.pstatic.net/cmstatic/desktop/v${version}/BAND-${version}-amd64.deb"
+checksum=88bbc33c1d831e51abad81cb43d4622382e3cb40de38f35d549effc979ea1f8c
+repository=nonfree
+restricted=yes
+do_extract() {
+	ar x ${XBPS_SRCDISTDIR}/BAND-${version}/BAND-${version}-amd64.deb
+	tar -xf data.tar.xz
+	gzip -d usr/share/doc/band/changelog.gz
+}
+do_install() {
+	vmkdir /opt/BAND
+	vcopy opt/BAND/* /opt/BAND
+	vinstall usr/share/applications/band.desktop 644 usr/share/applications
+	#install icons
+	for size in 16 32 48 64 128 256 512; do
+		#Insure containing directory
+		mkdir -p ${DESTDIR}/usr/share/icons/hicolor/${size}x${size}/apps
+		vinstall usr/share/icons/hicolor/${size}x${size}/apps/band.png 644 /usr/share/icons/hicolor/${size}x${size}/apps
+	done
+}
+post_install() {
+	vdoc usr/share/doc/band/changelog
+	vlicense opt/BAND/resources/LICENSE
+}
```

Comment:
```suggestion
do_install() {
	vmkdir opt
	vcopy opt/BAND opt

	vinstall usr/share/applications/band.desktop 0644 usr/share/applications
	vcopy usr/share/icons usr/share

	vmkdir usr/bin
	ln -s /opt/BAND/band ${DESTDIR}/usr/bin/band
}
post_install() {
	vlicense opt/BAND/resources/LICENSE
}
```
I've simplified what you wrote a bit. The icons are already in the proper structure, no need to manually do it :).

I'm not convinced it's necessary to include that changelog (if it isn't, the gzip line in extract isn't necessary).